databases/week1/db-lesson-1.sql

```diff
@@ -0,0 +1,58 @@
+
+-- Find out how many tasks are in the task table
+SELECT count(id)
+AS 'number of tasks'
+FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT count(id)
+FROM task
+WHERE due_date
+is NULL;
+
+-- Find all the tasks that are marked as done
+SELECT count(id)
+AS 'number of completed tasks'
+FROM task;
```

Comment:
Here you are finding the count of the total tasks as Q # 1, which is 35. Just to let you know, you already answered this in # 1. You'll need to find the tasks that are already done not to count how many they are. You can try it like this...& you can see if it makes sense.
```
SELECT task.id,title,due_date, status.name 
FROM task
JOIN status ON task.status_id=status.id
WHERE status.name="Done";
```